lib/contracts/message-reaction.ts

```diff
@@ -0,0 +1,36 @@
+import type { ContractDefinition } from '@balena/jellyfish-types/build/core';
+import { contractMixins } from 'autumndb';
+
+export const messageReaction: ContractDefinition = {
+	slug: 'message-reaction',
+	type: 'type@1.0.0',
+	name: 'message reaction',
+	markers: [],
+	data: {
+		schema: {
+			type: 'object',
+			required: ['version', 'data'],
+			properties: {
+				version: {
+					type: 'string',
+					const: '1.0.0',
+				},
+				data: {
+					type: 'object',
+					properties: {
+						actor: {
```

Comment:
I think it would make more sense to connect the actor to the reaction using a link instead of storing it directly into the reaction contract.